lib/ClangImporter/ImportDecl.cpp

```diff
@@ -9165,6 +9165,12 @@ void ClangImporter::Implementation::swiftify(AbstractFunctionDecl *MappedDecl) {
   if (!ClangDecl)
     return;
 
+  // FIXME: for private macro generated functions we do not serialize the
```

Comment:
We should tie FIXMEs to an issue so it actually gets done

Reply:
It's an interesting policy that we don't allow radars in comments, but do allow only tracking the issues internally. But if that's the case, then fair enough